sklearn/linear_model/tests/test_ridge.py

```diff
@@ -487,6 +487,35 @@ def test_class_weights():
     assert_array_almost_equal(clf.intercept_, clfa.intercept_)
 
 
+def test_class_weight_vs_sample_weight():
```

Comment:
I'd wondered whether it was copied from elsewhere. That's upsetting.
Perhaps we need an issue to track tests that should be factored into
common...

On 10 June 2015 at 14:28, Trevor Stephens <notifications@github.com> wrote:

> In sklearn/linear_model/tests/test_ridge.py
> <https://github.com/scikit-learn/scikit-learn/pull/4838#discussion_r32087333>
> :
>
> > @@ -487,6 +487,35 @@ def test_class_weights():
> >      assert_array_almost_equal(clf.intercept_, clfa.intercept_)
> >
> >
> > +def test_class_weight_vs_sample_weight():
>
> Agreed @jnothman <https://github.com/jnothman> , nothing special to Ridge
> here, purely a test for classifiers with class_weight (constructor) and
> sample_weight (fit) to check equivalency and multiplicative combination.
> Should be able to work its way into a common test I would think. This test
> appears in the tree tests (though with a multi-output additional step_ and
> elsewhere (I think I may have pilfered or altered it from SGD originally).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scikit-learn/scikit-learn/pull/4838/files#r32087333>.
>